app/connectors/connectors/sql/sql-common/pom.xml

```diff
@@ -55,8 +55,9 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>org.json</groupId>
-      <artifactId>json</artifactId>
+      <groupId>com.vaadin.external.google</groupId>
+      <artifactId>android-json</artifactId>
+      <version>0.0.20131108.vaadin1</version>
```

Comment:
Why is this lib better ? Version number looks scary (major and datewise)

Reply:
BTW, I was and always am a big fan of json-simple (only dep of Jolokia)

Never let me down ...

Reply:
Not better, just under a different license, JSONAssert uses this implementation of `org.json` which is a clean room implementation there is a bit of text on this in 81102c278ad47c76e2f71c6a0fb750ad3d9a9edc. Without this we have duplicate classes on classpath.

Reply:
JSONAssert is only for testing, or ? Is it used also during runtime ? wonder why jsonassert dep is included not only for scope test.